Source/Project64/N64 System/Mips/Pif Ram.cpp

```diff
@@ -297,19 +297,16 @@ void CPifRam::SI_DMA_READ()
 			{
 				continue;
 			}
-			RDRAM[RdramPos ^3] = m_PifRam[count];
+			RDRAM[BES(RdramPos)] = m_PifRam[count];
 		}
 	}
 	else
 	{
-		for (size_t i = 0; i < 64; i += 4)
-		{
-			unsigned __int32 pif_ram_dword;
-			std::memcpy(&pif_ram_dword, &PifRamPos[i], sizeof(unsigned __int32));
-
-			pif_ram_dword = swap32by8(pif_ram_dword);
+		register size_t i;

```

Comment:
There's no particular reason to leave this in a scope it's not used in. By the way, have you benchmarked or checked that the assembly produced is actually better with the register keyword. I can bet it does not alter semantics at all (it's also deprecated).